docs/deploy.md

```diff
@@ -1,108 +1,75 @@
-| dir| String | 1.0 | Location of the directory where are the applications to be deployed.| Yes, only when multiples files will be deployed and `file` is not specified.|
-| include| String | 1.0 | Comma or space-separated list of patterns of files that must be included. All files are included when this attribute is omitted.| No |
-| exclude| String | 1.0 | Comma or space-separated list of patterns of files that must be excluded. No files are excluded when this attribute is omitted.| No |
+The `deploy` task can be configured using the `deploy` block. Below are the properties that can be set in the `deploy` block.
 
-### Examples
+| Attribute | Type  | Since | Description | Required |
+| --------- | ----- | ----- | ----------- | -------- |
+| apps | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `apps` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
+| dropins | Object[] | 3.0 | Specifies which tasks/files should be deployed to the `dropins` directory. Applications can be passed in as the task that generates the file or as an application file. | No |
 
-1. Deploys a single file.
-  ```groovy
-    apply plugin: 'liberty'
+In addition to the `deploy` block, the `stripVersion` and `looseApplication` properties are used for application installation. More information on these properties can be found in the [Liberty server configuration](libertyExtensions.md#liberty-server-configuration).
 
-    liberty {
-        installDir = 'c:/wlp'
+Loose Application is only supported by Gradle 4.0 or higher for tasks of type ear.
 
-        server {
-            name = 'myServer'
+If you target an application to the `dropins` folder, no configuration is needed to get the server to start the application. When you want to configure an application in server.xml, the `location` is relative to the `apps` folder. Applications in the `apps` folder are not started without configuration. If you configure an application to install to the `apps` folder and it is not configured, default configuration is created in the `configDropins` folder.
 
-            deploy {
-                file = 'c:/files/app.war'
-            }
-        }
-    }
-  ```
+Multiple applications can be installed to the `apps` or `dropins` directories. This can be done by adding application files, or the tasks that generate these files, to the respective list.
 
-2. Deploy multiple files. Specifically, deploy `app.war` and `sample.war` but exclude `test-war.war`.
-  ```groovy
-    apply plugin: 'liberty'
+### Using the libertyApp configuration
 
-    liberty {
+The `libertyApp` configuration can be used to pull in application artifacts as dependencies and then install them to the server's `apps` directory. You can specify the artifact as any type of object that can be resolved in to a Dependency object by Gradle. The application artifact still needs to be a supported application type of `war` or `ear` to be installed.
```

Comment:
"The `libertyApp` dependency configuration...."

Is this documented somewhere else? Is this new?

Reply:
It was in the `installApps` documentation.